code/game/gamemodes/cult/blood_magic.dm

```diff
@@ -848,8 +848,12 @@
 			uses += max(1, temp)
 
 /obj/item/melee/blood_magic/manipulator/attack_self(mob/living/user)
-	var/list/options = list("Blood Orb (50)", "Blood Recharge (75)", "Blood Spear (150)", "Blood Bolt Barrage (300)")
-	var/choice = input(user, "Choose a greater blood rite...", "Greater Blood Rites") as null|anything in options
+	var/list/options = list("Blood Orb (50)" = image(icon = 'icons/obj/cult.dmi', icon_state = "summoning_orb"),
+							"Blood Recharge (75)" = image(icon = 'icons/obj/items.dmi', icon_state = "disintegrate" ),
```

Comment:
```suggestion
							"Blood Recharge (75)" = image(icon = 'icons/obj/items.dmi', icon_state = "disintegrate"),
```

Reply:
This could probably be 
 `"Blood Recharge (75)" = /obj/item/melee/blood_magic/empower` or whatever the path is. Right?

Reply:
It could be yeah, since the item has the sprite you want already and the radial menu will use that.
It's probably better to just leave it with the `image()` thing though, for consistency with the others.

Reply:
Blood recharge uses the sprite of a recoloured einath so using the path is probably clearer of what it is because it will also be recoloured.

Reply:
fixed